### PR TITLE
Update queries for sheet 100

### PR DESCRIPTION
**Pull request by esirK**
## Description
Fixes sheet 100 queries

Fixes # (issue)

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

## Screenshots
Screenshot 2021-02-25 at 18 43 29

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: reports/report_builder.py [hunk 1]
```diff
@@ -3032,23 +3032,19 @@ def ws_98(self, ws):
...
+                rows = model.objects\
+                        .values('covid19', 'topic')\
+                        .annotate(n=Count('id'))
+                for r in rows:
+                    covid19 = 'Y' if r['covid19'] == 1 else 'N'
+                    counts.update({(covid19, TOPIC_GROUPS[r['topic']]): r['n']})
+                secondary_counts[media_type] = counts
     
-        self.tabulate(ws, counts_list[0], TM_MEDIA_TYPES, MAJOR_TOPICS, row_perc=True)
-        c = ws.dim_colmax + 2
```
Comment (kilemensi) [inline reports/report_builder.py:3050]: We no longer do this littler traditional / digital media columns separation?

**Review by kilemensi — APPROVED** [review]
🚀  @esirK

**Comment by kilemensi** [comment]
PS @esirK, is this query defined for all 3 types of reports i.e. global, region and country?

**Comment by esirK** [comment]
> PS @esirK, is this query defined for all 3 types of reports i.e. global, region and country?

Right now it's just global. Lemme add region and country

File: reports/report_builder.py (hunk 1 above)
Comment (esirK) [reply reports/report_builder.py:3050]: Naah. `tabulate_secondary_cols` takes care of that.

**Pull request merged**